### ISSUE TITLE
UI Homepage. Settings homepage

### ISSUE THREAD

**Issue by darint916** [sqz269/music-player#28]
The homepage setting is not selected to current when open; changing it doesn't do anything now. Albums per page in pagination not working

**Comment by sqz269** [comment]
Yeah, it's a work in progress. Probably going to be one of the last things on my to-do list.
